roles/welcome/tasks/main.yml

```diff
@@ -15,11 +15,6 @@
     src: "site/index.html"
     dest: "/var/www/welcome/index.html"
 
-- name: copy the openconext_ca certificate
-  copy:
-    src: "{{ inventory_dir }}/files/certs/{{ tls_ca }}"
-    dest: "/var/www/welcome/openconext_ca.crt"
```

Comment:
Still linked from index.html

Reply:
Fixed in 5a17d7943f08986ec2f3665dee2260687405eced